header.php

```diff
@@ -52,11 +52,13 @@
 
   $logo_id = IGV_get_option('_igv_social_options', '_igv_metadata_logo_id');
```

Comment:
yh thats why i updated the boiler to what it is now so we dont have to remember to correctly add the data rather put it in the end users' hands.